packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1442,4 +1442,46 @@
     <protected>false</protected>
     <shortDescription>Show less</shortDescription>
   </labels>
+  <labels>
+    <fullName>quantic_SearchFieldWithSuggestions</fullName>
+    <value>Search field with suggestions. Suggestions may be available under this field. To send, press Enter.</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Search field with suggestions.</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_SuggestionFound</fullName>
+    <value>{{0}} search suggestion found, to navigate use up and down arrows.</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>1 search suggestion found</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_SuggestionFound_plural</fullName>
+    <value>{{0}} search suggestions found, to navigate use up and down arrows.</value>
```

Comment:
What do you think of a seperate label for `to navigate use up and down arrows.` ? Would we ever want to use the first part of the label without the navigation sentence?

Reply:
I would do this separation when we face the need of using one part separately, cause it's very possible that there will be no case where we will  need that.